docs/introduction-to-smart-contracts.rst

```diff
@@ -256,14 +257,17 @@ Ethereum this is roughly every 17 seconds.
 
 As part of the "order selection mechanism" (which is called "mining") it may happen that
 blocks are reverted from time to time, but only at the "tip" of the chain. The more
-blocks that are added on top, the less likely it is. So it might be that your transactions
+blocks that are added on top, the less likely this block will be reverted. So it might be that your transactions
```

Comment:
``The more blocks are added on top of a particular block, the less likely this block will be reverted``?